server/src/main/java/keywhiz/KeywhizService.java

```diff
@@ -184,4 +193,7 @@ public static ObjectMapper customizeObjectMapper(ObjectMapper objectMapper) {
     objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
     return objectMapper;
   }
+
+  @Override
+  public void onFatalError() {}
```

Comment:
Normally this method issues a System.exit(1), which is a pretty harsh way to exit the program. Doing nothing allows the JVM to exit a bit more gracefully.